IVRv2/app/fsm/fsm.py

```
@@ -152,59 +226,150 @@ async def get_next_actions(self, input_: str, ivr_state_doc: IVRCallStateMongoDo
+
+        return prepared_actions
+
+
+    async def _stop_websocket_audio_for_state(
+        self, current_state, conversation_id: str
+    ) -> None:
+        """Stop WebSocket audio playback using control WebSocket connection."""
+        try:
+            ws_service = await get_websocket_service()
+            await ws_service.stop_audio(conversation_id)
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)